packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/components/ColumnComponent.java

```diff
@@ -61,12 +62,12 @@ public Element render() {
         return new Fragment(fragmentProps);
     }
 
-    private Element doRender(VariableManager variableManager, Object object, int index) {
+    private Element doRender(VariableManager variableManager, Object object, int columnIndex) {
         ColumnDescription columnDescription = this.props.columnDescription();
 
         VariableManager columnVariableManager = variableManager.createChild();
         columnVariableManager.put(VariableManager.SELF, object);
-        columnVariableManager.put("columnIndex", index);
+        columnVariableManager.put("columnIndex", columnIndex);
```

Comment:
Just a sidenote, the index given to everybody is not the right index anymore since it gets changed later down.